Bottom: 391df273cf6a38233c8bacdfa84c456f50fcd534
Top:    1ece294614eede2ff73f68d0f1850bcbc6f467a8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 18:52:08 +0000

Refresh of djopt-c-if-0-out-an-unused-sta

---

diff --git a/src/djopt.c b/src/djopt.c
index 800e574..b5d9c39 100644
--- a/src/djopt.c
+++ b/src/djopt.c
@@ -2491,6 +2491,7 @@ dump_all ()
 }
 #endif
 
+#if 0
 static void
 nudge_corner (corner_s * c, int dx, int dy, corner_s * prev_corner)
 {
@@ -2511,6 +2512,7 @@ nudge_corner (corner_s * c, int dx, int dy, corner_s * prev_corner)
 	nudge_corner (oc, 0, dy, c);
     }
 }
+#endif
 
 static line_s *
 choose_example_line (corner_s * c1, corner_s * c2)
